app/routes/events/components/EventEditor/index.js

```diff
@@ -88,10 +88,13 @@ function EventEditor({
   const isEditPage = eventId !== undefined;
 
   const [useMazemap, setUseMazemap] = useState<boolean>(false);
+  const [hasFeedbackQuestion, setHasFeedbackQuestion] =
+    useState<boolean>(false);
 
   useEffect(() => {
     if (initialized) {
       setUseMazemap(event?.mazemapPoi?.value > 0);
+      setHasFeedbackQuestion(!!event?.feedbackDescription);
     }
     // Should only run once when initialized to initialize useMazemap state
     // eslint-disable-next-line react-hooks/exhaustive-deps, react-app/react-hooks/exhaustive-deps
```

Comment:
Would be nice if this wasn't necessary, as it's not done this way for other boolean values in events.

Other than that LGTM:)

Reply:
I figured out how to do it without the `useState` 😁